cmd/minishift/cmd/addon/addon_apply.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addon
+
+import (
+	"fmt"
+
+	"github.com/docker/machine/libmachine"
+	"github.com/docker/machine/libmachine/provision"
+	"github.com/minishift/minishift/cmd/minishift/cmd/util"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	minishiftConfig "github.com/minishift/minishift/pkg/minishift/config"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	routingSuffix = "routing-suffix"
+)
+
+var addonsApplyCmd = &cobra.Command{
+	Use:   "apply ADDON_NAME",
+	Short: "Executes the specified add-on.",
+	Long:  "Executes the specified add-on",
+	Run:   runApplyAddon,
+}
+
+func init() {
+	AddonsCmd.AddCommand(addonsApplyCmd)
+}
+
+func runApplyAddon(cmd *cobra.Command, args []string) {
+
+	if len(args) == 0 {
+		atexit.ExitWithMessage(1, emptyAddOnError)
+	}
+
+	addonName := args[0]
```

Comment:
If we do this in a loop, treating each argument as a add-on name, we can apply multiple add-ons in one call. 

The same should then be applied to 'enable', 'disable' and 'install'. This will improve the general usability of the add-on commands.